boxsdk/util/compat.py

```diff
@@ -1,106 +0,0 @@
-# coding: utf-8
-
-from __future__ import absolute_import, division, unicode_literals
-
-import six
-from six.moves import map
-
-
-NoneType = type(None)
-
-
-def with_metaclass(meta, *bases, **with_metaclass_kwargs):
-    """Extends the behavior of six.with_metaclass.
```

Comment:
This is not needed anymore
Read: https://stackoverflow.com/questions/18513821/python-metaclass-understanding-the-with-metaclass